projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.ts

```diff
@@ -55,7 +55,10 @@ export class DataLayerManagerService {
   private timelineRangeSubject = new ReplaySubject<{ min: number; max: number }>();
   timelineRange$ = this.timelineRangeSubject.pipe(throttleTime(100, undefined, { leading: true, trailing: true }));
 
+  loader$ = new BehaviorSubject<boolean>(false); 
   loading = false;
+  loadingObservable = new BehaviorSubject<boolean>(false);
```

Comment:
why do we have two subjects for the same thing? I prefer "loading" over "loader"

Reply:
Yes ,I am forgot to remove that lodingObservable that was previously use.